docs/pages/setup/reference/cli.mdx

```diff
@@ -1310,6 +1310,342 @@ $ tctl auth rotate --type=user --grace-period=200h
+
+#### Examples
+
+Test the auth connector from `connector.yaml`:
+
+```code
+$ tctl sso test connector.yaml
+```
+
+The command is designed to be used in conjunction with `tctl sso configure` family of commands:
+
+```code
+$ tctl sso configure ... | tctl sso test
+```
+
+The pipeline may also utilize `tee` to capture the connector generated with `tctl sso configure`. This will save the connector in `connector.yaml`:
+
+```code
+$ tctl sso configure ... | tee connector.yaml | tctl sso test
+```
+
+You can test an existing auth connector by combining the command with `tctl get`:
+
+```bsh
+$ tctl get saml/your-connector-name --with-secrets | tctl sso test
+````
+
+<Admonition
+  type="note"
+  title="Important"
+>
+  Make sure to include `--with-secrets` flag, or the exported auth connector will not be testable.
+</Admonition>
+
+### tctl sso configure github
+
+Configure GitHub auth connector.
+
+Required params `--id` and `--secret` come from the [Github OAuth app](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app).
+
+The flag `--teams-to-roles` can be provided multiple times to specify which Github Teams are assigned which roles.
+
+```code
+$ tctl sso test --id=GITHUB_CLIENT_ID --secret=GITHUB_SECRET --teams-to-roles=octocats,devs,access,editor [<other-flags>]
+```
+
+#### Arguments
+
+This command accepts no arguments.
+
+#### Flags
+
+Mandatory flags: `--id`, `--secret`, `--teams-to-roles`.
+
+| Name                      | Default Value(s) | Allowed Value(s)           | Description                                             |
+|---------------------------|------------------|----------------------------|---------------------------------------------------------|
+| `-n`, `--name`            | `github`         | resource name              | Connector name.                                         |
+| `-r`, `--teams-to-roles`  | none             | `org,team,role1,role2,...` | Sets teams-to-roles mapping. Repeatable.                |
+| `--display`               | none             | display name               | Controls how this connector is displayed.               |
+| `--id`                    | none             | Github OAuth2 client id    | Github app client ID.                                   |
+| `--secret`                | none             | Github OAuth2 secret       | Github app client secret.                               |
+| `--redirect-url`          | none             | Valid callback URL.        | Authorization callback URL.                             |
+| `--ignore-missing-roles`  |                  |                            | Ignore missing roles referenced in `--teams-to-roles`.  |
+
+#### Global flags
+
+These flags are available for all commands `--debug, --config`. Run
+`tctl help <subcommand>` or see the [Global Flags section](#tctl-global-flags).
+
+#### Examples
+
+Generate Github auth connector. Two role mappings are defined:
```

Comment:
```suggestion
Generate GitHub auth connector. Two role mappings are defined:
```

Try to use `GitHub` as I just opened a bunch of PRs to change this everywhere :)